src/test/java/org/folio/marc4ld/mapper/MarcBib2LdMapperIT.java

```diff
@@ -283,12 +254,6 @@ void map_shouldReturnCorrectlyMappedResource() {
     validateTitle3(edgeIterator.next());
     validateVariantTitle(edgeIterator.next());
     validateParallelTitle(edgeIterator.next());
-    validateProviderEvent(edgeIterator.next(), PE_MANUFACTURE, "Manufacture261");
```

Comment:
We already have separate tests for MARC fields 26X. So, these tests are not needed any more. Hence removing them.